paccmann_predictor/models/bimodal_mca.py

```diff
@@ -79,52 +78,65 @@ def __init__(self, params, *args, **kwargs):
         self.params = params
         self.smiles_padding_length = params['smiles_padding_length']
         self.protein_padding_length = params['protein_padding_length']
-
+        self.encoding = params['encoding']
```

Comment:
If this parameter is absolutely needed, it should be added to the docstring. In general, it would be better to work with defaults and the `.get` method.

Reply:
Also, to which input branch does this refer?